ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx

```diff
@@ -31,55 +31,40 @@ type State = {
 const Root = styled.div`
   position: relative;
   overflow: none;
-  padding: 0;
   margin: 0;
-  display: flex;
+  white-space: nowrap;
+  background-color: ${props => props.theme.primaryColor};
 `
 const Menu = styled.div`
   position: absolute;
   top: ${props => props.theme.minimumButtonSize};
-  padding: 0;
+  padding-top: ${props => props.theme.minimumSpacing};
+  padding-bottom: ${props => props.theme.minimumSpacing};
   margin: 0;
-  width: 100%;
   overflow: auto;
   z-index: ${props => props.theme.zIndexDropdown};
   background-color: ${props => props.theme.backgroundDropdown};
+  opacity: 1 !important;
+  border: 1px solid ${props => props.theme.backgroundModal};
 `
 
 const DefaultButton = styled.button`
-  flex: 1;
+  padding-right: ${props => props.theme.largeSpacing};
+  padding-left: ${props => props.theme.largeSpacing};
```

Comment:
Same sort of thing here:
```suggestion
  padding: 0 ${props => props.theme.largeSpacing} 0 ${props => props.theme.largeSpacing};
```

Reply:
IMO the explicit attributes are more readable if only two sides are being set